plugins/redshift/dbt/adapters/redshift/connections.py

```diff
@@ -87,10 +93,21 @@ def fresh_transaction(self, name=None):
 
     @classmethod
     def fetch_cluster_credentials(cls, db_user, db_name, cluster_id,
-                                  duration_s, autocreate, db_groups):
+                                  iam_profile, duration_s, autocreate,
+                                  db_groups):
         """Fetches temporary login credentials from AWS. The specified user
         must already exist in the database, or else an error will occur"""
-        boto_client = boto3.client('redshift')
+
+        if iam_profile is None:
+            boto_client = boto3.client('redshift')
+        else:
+            logger.debug("Connecting to Redshift using 'IAM'" +
+                         f"with profile {iam_profile}")
+            boto_session = boto3.Session(
+                profile_name=iam_profile,
+                region_name='eu-west-1'
```

Comment:
I think we may not want to hard-code this region name here. Do you think this should be an option in the profile? Or even better, can we document that the region should be specified in the configured AWS IAM profile?

Reply:
Ops, my bad, I didn't mean to add that, just pushed the fix. Yea, I think everything should be specified in the profile.

Is there a repo for the user documentation? Couldn't find anything in the dbt repo

Reply:
The documentation is maintained in this repository: https://github.com/fishtown-analytics/docs.getdbt.com/